Podfile

```diff
@@ -11,9 +11,8 @@ def wordpress_authenticator_pods
   ##
   pod 'Gridicons', '~> 1.0'
   pod 'WordPressUI', '~> 1.7.0'
-  pod 'WordPressKit', '~> 4.0-beta.0' # Don't change this until we hit 5.0 in WPKit
-  pod 'WordPressShared', '~> 1.9-beta' # Don't change this until we hit 2.0 in WPShared
-
+  pod 'WordPressKit', '~> 4.14-beta'
+  pod 'WordPressShared', '~> 1.10-beta'
```

Comment:
This isn't necessary. We update WordPressKit and WordPressShared in the host app's podfile. So for example, Authenticator and WordPress-iOS can have the latest pods by running `bundle exec pod update WordPressKit` (in each repo) and it will update to 4.14-beta.

Reply:
Rolled back!